src/registrar/templatetags/custom_filters.py

```diff
@@ -48,3 +51,18 @@ def contains_checkbox(html_list):
         if re.search(r'<input[^>]*type="checkbox"', html_string):
             return True
     return False
+
+
+@register.filter
+def get_organization_long_name(organization_type):
+    # https://gist.github.com/OmenApps/3eef60ba4204f3d1842d9d7477efcce1#file-django_choices-txt-L28
+    organization_choices_dict = dict(
+        DomainApplication.OrganizationChoicesVerbose.choices
+    )
+
+    long_form_type = organization_choices_dict[organization_type]
+    if long_form_type is None:
+        logger.error("Organization type error, triggered by a template's custom filter")
```

Comment:
(optional) We can afford to be more specific here given that this log is for internal uses. If the almighty linter allows, I'd recommend including some metadata about why the error occurred (basically: "long_form_type was None" or "org type cannot be none")